index.html

```diff
@@ -62,5 +62,198 @@ <h1>Hi👋, I'm Cephas Tokoli</h1>
          </div>     
         </div>   
     </section>
+   <section class="grid-container">
+            <section class="project-section">
+                <div>
+                    <h2 class="headings">Projects</h2>
+                </div>
+        
+                <div class="project">
+                    <div class="laptop-img">
+                        <img src="images/laptop.png" alt="laptop">
+                    </div>
+                    <div class="project-details">
+                        <div class="tech-details">
+                            <h2 class="project-name">Project name goes here</h2>
+                        <div class="tech-stack">
+                            <p>HTML/CSS</p>
+                            <p>Ruby on Rails</p>
+                            <p>JavaScript</p>
+                        </div>
+                        <div class="details-button">
+                        <div>
+                            <p >See this project</p>
+                            <img src="images/Union.svg" alt="union">
+                        </div>
+                        </div>
+                        </div>
+                    </div>
+                </div>
```

Comment:
As requested by the previous reviewer, please kindly use `Flexbox` to lay out an individual card. 🛠💪

Reply:
Thanks for the review @abdulhamiid, I am not clear on this direction. I am really confused about where to apply that CSS rule. Use Flexbox to lay out an individual card, as in applying Flexbox to each project or how exactly.
```suggestion
                <div class="project">
                    <div class="laptop-img">
                        <img src="images/laptop.png" alt="laptop">
                    </div>
                    <div class="project-details">
                        <div class="tech-details">
                            <h2 class="project-name">Project name goes here</h2>
                        <div class="tech-stack">
                            <p>HTML/CSS</p>
                            <p>Ruby on Rails</p>
                            <p>JavaScript</p>
                        </div>
                        <div class="details-button">
                        <div>
                            <p >See this project</p>
                            <img src="images/Union.svg" alt="union">
                        </div>
                        </div>
                        </div>
                    </div>
                </div>
```

Reply:
> Thanks for the review @abdulhamiid, I am not clear on this direction. I am really confused about where to apply that CSS rule. Use Flexbox to lay out an individual card, as in applying Flexbox to each project or how exactly.

Hi @ctokoli, looking through your markup, the container of each project card is:

```HTML
<div class="project">
 . . .
</div>
```

So kindly use CSS Flexbox `display: flex` to layout individual cards. 🛠👍